volatility3/cli/volshell/windows.py

```diff
@@ -44,11 +44,58 @@ def list_processes(self):
             )
         )
 
+    def get_process(self, pid=None, v_offset=None, p_offset=None):
+        """Returns the EPROCESS object that matches the pid. If v_offset/p_offset is provided, construct the EPROCESS object at the provided address. Only one parameter is allowed."""
+
+        if sum(1 if x is not None else 0 for x in [pid, v_offset, p_offset]) != 1:
+            print("Only one parameter is accepted")
+            return None
+
+        kernel_name = self.config["kernel"]
+        kernel = self.context.modules[kernel_name]
+
+        kernel_layer_name = kernel.layer_name
+
+        kernel_layer = self.context.layers[kernel_layer_name]
+        memory_layer_name = kernel_layer.dependencies[0]
```

Comment:
Again, likely to be right and until we get a proper API for what people generally mean when they say "physical layer" this'll do, but it's possibly it could be a swap layer or something entirely different one day.  Sadly, the question's not an easy one to answer, so for now this is ok (and much better than hardcoding it with a specific name!).  5:)